workers_test.go

```diff
@@ -165,6 +165,18 @@ const (
   "success": true,
   "errors": [],
   "messages": []
+}`
+	attachWorkerToDomainResponse = `{
+    "result": {
+        "id": "e7a57d8746e74ae49c25994dadb421b1",
+	"zone_id":"foo",
```

Comment:
you'll want to use `testZoneID` here instead.

Reply:
Need to move out of the const block in order to done fmt.Sprintf